### PR TITLE
Updated the dev vm to use the bpfd installer

### PR DESCRIPTION
**Pull request by nerdalert**
Signed-off-by: Brent Salisbury <bsalisbu@redhat.com>

### CODE REVIEW

**Comment by nerdalert** [comment]
This is dependent on #144 and #139

To spin up the VM:

```
cd bpfd/packaging/vm-deployment/
vagrant up
vagrant ssh
sudo systemctl status bpfd

● bpfd.service - Run bpfd as a service
     Loaded: loaded (/usr/lib/systemd/system/bpfd.service; static)
     Active: active (running) since Tue 2022-09-13 15:58:43 UTC; 7min ago
   Main PID: 29470 (bpfd)
      Tasks: 5 (limit: 4664)
     Memory: 27.5M (low: infinity)
        CPU: 15ms
     CGroup: /system.slice/bpfd.service
             └─ 29470 /usr/sbin/bpfd

Sep 13 15:58:43 fedora36.localdomain systemd[1]: Started bpfd.service - Run bpfd as a service.
Sep 13 15:58:43 fedora36.localdomain bpfd[29470]: Log using journald
Sep 13 15:58:43 fedora36.localdomain bpfd[29470]: Listening on [::1]:50051
```

**Comment by nerdalert** [comment]
Added a readme @astoycos, ty for taking a look!

**Comment by astoycos** [comment]
Didn't get through testing this since it's not supported with libvert but I'd imagine this would mainly be used by developers on other OS's like Mac's and Window's

Thanks for adding the readme @nerdalert!

**Review by astoycos — APPROVED** [review]

**Comment by astoycos** [comment]
Note for onlookers the systemd service file(deleted here) was updated and moved to https://github.com/redhat-et/bpfd/tree/main/scripts

File: packaging/vm-deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- Vagrant - Vagrant deploys the OS and will trigger the Anisble playbook. See [Vagrant Downloads](https://www.vagrantup.com/docs/installation) and choose your target OS.
+- Ansible - Once provisioned, Ansible will configure the VM. See [Installing Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) and choose your target OS.
+
+### Deploy the VM
+
+Once the dependencies are installed, simply clone bpfd and run the following commands:
+
+```console
+# Clone the bpfd repo:
+$ git clone https://github.com/nerdalert/bpfd.git
```
Comment (Billy99) [inline packaging/vm-deployment/README.md:17]: Did you intend to leave `nerdalert` here? Should this be `redhat-et`?

Comment (astoycos) [reply packaging/vm-deployment/README.md:17]: @nerdalert Can you fix this RQ :)

Comment (nerdalert) [reply packaging/vm-deployment/README.md]: lol ty

**Pull request merged**